lib/runCode.js

```diff
@@ -214,7 +213,7 @@ module.exports = function (opts, cb) {
   }
 
   function parseVmResults (err) {
-    err = runState.vmError || err
+    err = runState.vmError || (err && (err.error || err))
```

Comment:
Would be nice if the instances not returning an error object would be fixed instead. Do you happen to know which ones?

Reply:
Okay.  I was able to remove this.  now we are back to the failures that @Silur mentioned.